src/to-have-style.js

```diff
@@ -20,6 +20,7 @@ function isSubset(styles, computedStyle) {
     Object.entries(styles).every(
       ([prop, value]) =>
         computedStyle[prop] === value ||
+        computedStyle.getPropertyValue(prop) === value ||
         computedStyle.getPropertyValue(prop.toLowerCase()) === value,
```

Comment:
> Can you make it so that we keep the lower case transformation when the prop does not look like a custom css prop?

After reading the whole discussion [here](https://github.com/testing-library/jest-dom/pull/154#discussion_r345172964), I thought this would be the least disruptive way.

Reply:
I still think that we may need to detect specifically if the prop name corresponds to a custom CSS property. They are case-insensitive. The way the code is right now, if I have an element with `--var-name: 0px`, a test checking for `--VAR-NAME: 0px` would pass, and that's not accurate.

Can't you do something along the lines of:

```js
const isCustomProp = prop.startsWith('--')
if (!isCustomProp) {
  prop = prop.toLowerCase()
}
// Keep the condition as before
```

Reply:
see my latest commit